### ISSUE TITLE
The case of making a phone call.

### ISSUE THREAD

**Issue by vopbs** [tayler6000/pyVoIP#115]
I hope we can make a case of making a phone call. I want to turn it into a test phone so that we can call and test something ourselves, okay?

**Comment by jrozhon** [comment]
yes, this is exactly the scenario I am using this library for. But still, keep in mind that this is not a complete SIP implementation, so many cases would require you to extend this library by yourself to meet your needs. For me, I had to add sending the DTMF codes based on a API call, for example.

Regards, J
